pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -125,8 +125,7 @@ const (
 )
 
 func (c *Cluster) makeJob(osdProps osdProperties, provisionConfig *provisionConfig) (*batch.Job, error) {
-	osdOnPVC := osdProps.pvc.ClaimName != ""
-	osdOnPVCMetadata := osdProps.metadataPVC.ClaimName != ""
+	osdOnPVC := osdProps.onPVC()
```

Comment:
Since we have a method now why not just use `osdProps.onPVC()`? Otherwise, it feels like this defeats the purpose of the methods if it's used in a variable.

Reply:
Indeed. Fixed.